src/responder/secrets/proxy.c

```diff
@@ -215,18 +231,33 @@ int proxy_sec_map_headers(TALLOC_CTX *mem_ctx, struct sec_req_ctx *secreq,
             }
         }
         if (forward) {
+            DEBUG(SSSDBG_TRACE_LIBS, "Forwarding header %s:%s\n",
+                                      secreq->headers[i].name,
+                                      secreq->headers[i].value);
+
```

Comment:
While still here, I'd prefer the 3rd and 4th arguments aligned to the first one. This comment applies for a few other parts of the code as well.